### DISCUSSION TITLE
Is there a way to sync titles between different stores?

### DISCUSSION THREAD

**Discussion by didiowen** [joonaspaakko/audible-library-extractor#149] [Q&A]
Hi everyone,

Excited to find this extension because I sometimes switch between US and UK stores to listen to specific titles that are not available in the other store, or just for a better price. As both of my libraries have 200+ titles and more in the wishlist, I would love to be able to compare or sync between these two libraries so that I won't buy a purchased title in the future or to renew my wishlist manually which is cumbersome...Any suggestions?

**Comment by joonaspaakko** [comment]
Even for manually comparing each library, there aren't really any great options in the extension. You might do better just opening One of the accounts normally and the other in incognito mode and searching each library individually on the Audible website. 

> The ability to at the very least view 2 data sets within one gallery could perhaps be added, but I make no promises right now. 
> 
> There's a big issue that takes precedence; you can't extract bigger libraries because half way through Audible blocks users ip address for 10 minutes.

Best you can probably do right now is install the extension into two different Chromium browsers and extract UK in one and US in the other, then maybe use the search in both...

An additional thing you could do is in the extension gallery `Extension tools > Export CSV` from both browsers and then combine them in a spreadsheet application and search both data sets at the same time.

> You can by the way [export raw data](https://joonaspaakko.gitbook.io/audible-library-extractor/gallery/import-export-data) which could be used to switch between two extracted libraries (more specifically their data sets) though somewhat clunkily, but perhaps that doesn't even help in your situation.
